controller/mentorTraningController.js

```diff
@@ -0,0 +1,105 @@
+const ZuriTrainingMentor = require('../models/ZuriTrainingMentorModel');
+const { responseHandler } = require('../utils/responseHandler');
+const { validationResult } = require('express-validator');
+
+module.exports = {
+    createApplication: async (req, res) => {
+        try{
+            const errors = await validationResult(req);
+
+            if (!errors.isEmpty()) {
+                responseHandler(res, { errors: errors.array() });
```

Comment:
this errors: errors.array() should be in data field and message to your and other field in your responseHandler